b3/sql/sqlite/b3-update-1.10.0.sql

```diff
@@ -4,6 +4,9 @@
 -- Add a table field for the xlr_playerstats table
 ALTER TABLE `xlr_playerstats` ADD `id_token` VARCHAR(10) NOT NULL DEFAULT '';
 
+-- Change xlr_actionstats so that one can supply an event of 26 characters
+ALTER TABLE  `xlr_actionstats` CHANGE  `name`  `name` VARCHAR( 26 )
```

Comment:
This won't work as SQLite does not support altering a column. But also [SQLite does not either enforce length on TEXT type columns](http://www.sqlite.org/faq.html#q9), so there is no need for an update sql file if it aims to just change a column length.